src/index.ts

```diff
@@ -1,5 +1,6 @@
+import { hash } from "bcrypt-ts";
 import fastify, { FastifyReply, FastifyRequest } from "fastify";
-import { PrismaClient } from "./generated/prisma/index.js";
+import { PrismaClient } from "../node_modules/.prisma/client/index.js";
```

Comment:
```suggestion
import { PrismaClient } from "@prisma/client";
```

Não dá pra importar desse caminho? É o padrão quando não tem campo output lá no `schema.prisma`.